doc/cifarSC2011/advanced_theano.txt

```diff
@@ -179,96 +180,200 @@ Inplace optimization
 Profiling
 ---------
 
-- To replace the default mode with this mode, use the Theano flags ``mode=ProfileMode``
+- To replace the default mode with this mode, use the Theano flags ``profile=True``
 
-- To enable the memory profiling use the flags ``ProfileMode.profile_memory=True``
+- To enable the memory profiling use the flags ``profile=True,profile_memory=True``
 
-Theano output:
```

Comment:
Now we don't put in Theano presentation, so this problem don't happen anymore. But I think as it is in Theano, we can update it. Or we should put a warning at the top that tell it isn't updated and point elsewhere.

As it was updated, I would keep it like this.